### PR TITLE
198 mssql

### LINKED ISSUE #198: MSSQL
**What application or protocol would you like to be added? Links to RFCs or other documentation can be helpful.**
MS SQL (Microsoft SQL Server) should be added as a protocol to be supported by Dynamicbeat. [A SQL driver](https://github.com/denisenkom/go-mssqldb) has been written to support connecting to MSSQL databases, which could be used to support the protocol.

**Please explain why this protocol cannot be monitored by the existing check types that Dynamicbeat supports.**
There is currently no support for traditional databases in Dynamicbeat (those that don't operate over simple HTTP).

### PR DESCRIPTION
**Pull request by oneNutW0nder**
Description
-----------

Please provide a summary of the changes this PR makes.

This PR implements a check for the `MSSQL` service.

Fixes #198 

## Merge Checklist

- [x] I have tested this change locally to make sure it works
- [x] I have updated the documentation as necessary
- [x] I have added a release note under the [Unreleased section of the Changelog](../CHANGELOG.md#unreleased)
- [x] Any relevant labels have been added
- [x] This PR is being merged into `dev`, unless it's a PR for a release

### CODE REVIEW

File: docs/reference/mssql.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+MSSQL
+=====
+
+| Name         | Type   | Required     | Description                                                          |
+| ------------ | ------ | ------------ | -------------------------------------------------------------------- |
+| Host         | String | Y            | IP or FQDN for the MySQL server                                      |
```
Comment (s-newman) [inline docs/reference/mssql.md:6]: Should change "MySQL" to "MSSQL"

**Review by s-newman — CHANGES_REQUESTED** [review]

File: docs/reference/mssql.md (hunk 1 above)
Comment (oneNutW0nder) [reply docs/reference/mssql.md]: Thanks, fixed!

**Review by s-newman — APPROVED** [review]
lgtm push to prod

**Comment by oneNutW0nder** [comment]
pushing to prod...

**Pull request merged**